storage/Devices/LuksImpl.cc

```diff
@@ -496,6 +541,10 @@ namespace storage
 	else
 	    size = 0 * B;
 
+	// If we use integrity, calculate a pesimistic size
+	if (!get_integrity().empty())
+	    size = static_cast<unsigned long long>(size * 0.85);
```

Comment:
Why not ```size *= 0.85```?

Reply:
To be more explicit in the type, but I will change it (not used to C++ idioms)

Reply:
Done